### PR TITLE
VNet integration document update

### PR DESCRIPTION
**Pull request by ritika-msft**

### CODE REVIEW

File: 01-Setup and getting started/README.md [hunk 1]
```diff
@@ -769,4 +796,13 @@ In this case, we recommend you to upgrade the Function app hosting plan (pricing
...
-  > _**Note:** In future if you run the AzTS installation command (`Install-AzSKTenantSecuritySolution`) to upgrade your existing AzTS setup, you will have to repeat the above steps._
\ No newline at end of file
+  > _**Note:** In future if you run the AzTS installation command (`Install-AzSKTenantSecuritySolution`) to upgrade your existing AzTS setup, you will have to repeat the above steps._
+<br> 
+
+  #### **Why should I integrate my AzTS solution setup with VNet?**
+
+1. Integrating AzTS solution with Vnet, ensures that all critical backend resources like - storage account, log analytics workspace, function apps etc. are not accessible over public network.
+
+2. It ensures that all inbound communication to backend resources are routed through private network (Vnet).
```
[the remark this answers is not in the document]
Comment (ritika-msft) [reply 01-Setup and getting started/README.md:805]: updated

File: 01-Setup and getting started/README.md [hunk 2]
```diff
@@ -298,6 +301,7 @@ $DeploymentResult = Install-AzSKTenantSecuritySolution `
                 -SendAlertNotificationToEmailIds @('<EmailId1>', '<EmailId2>', '<EmailId3>') `
                 -EnableAutoUpdater `
                 -EnableAzTSUI `
+                #-EnableVnetIntegration `
```
[the remark this answers is not in the document]
Comment (ritika-msft) [reply 01-Setup and getting started/README.md]: updated optional parameters as required

File: 01-Setup and getting started/README.md [hunk 3]
```diff
@@ -453,7 +462,8 @@ $HostSubscriptionId = "<HostSubscriptionId>"
 $HostResourceGroupName = "<HostResourceGroupName>"
 
 Start-AzSKTenantSecuritySolutionOnDemandScan -SubscriptionId $HostSubscriptionId `
-                                             -ScanHostRGName $HostResourceGroupName
+                                             -ScanHostRGName $HostResourceGroupName `
+                                             #-EnableVnetIntegration
```
[the remark this answers is not in the document]
Comment (ritika-msft) [reply 01-Setup and getting started/README.md]: updated optional parameter as required

File: 01-Setup and getting started/README.md [hunk 4]
```diff
@@ -472,6 +482,12 @@ The below steps will help you to verify and understand different resources and f
   ![Resources](../Images/12_TSS_Resource_Group_1.png)	
   ![Resources](../Images/12_TSS_Resource_Group_2.png)	
 
+  > **Note:** _If the AzTS Solution is integrated to VNet, verify that below additional resources also got created, along with the once shown above._
+
+  ![Resources](../Images/12_TSS_Resource_Group_3.png)
```
[the remark this answers is not in the document]
Comment (ritika-msft) [reply 01-Setup and getting started/README.md:486]: added border to images

File: 01-Setup and getting started/README.md [hunk 5]
```diff
@@ -516,6 +540,7 @@ The below steps will help you to verify and understand different resources and f
...
 |ATS_5_MGTreeProcessor| Responsible to fetch details about all the management group that has been granted access as Reader using central MI. All these management groups will be fetched by the job and persisted into LA. This function is disabled by default. To enable this function, you need to add/update ` FeatureManagement__ManagementGroups : true ` and `ManagementGroupConfigurations__ManagementGroupId : <Root_Management_Group_id> ` application setting on Azure Portal. To update settings, go to your App Service --> Configuration --> New application settings --> Save after adding/updating the setting.
+|ATS_7_InitiateOnDemandProcessing| Reads message from the AzTS storage queue and trigger functions (ATS_1_SubscriptionInvProcessor, ATS_2_BaselineControlsInvProcessor, ATS_3_SubscriptionRBACProcessor, ATS_4_WorkItemScheduler) of MetadataAggregator, if the AzTS solution is integrated to `VNet`.
```
[the remark this answers is not in the document]
Comment (ritika-msft) [reply 01-Setup and getting started/README.md]: rephrased the description

File: 01-Setup and getting started/README.md [hunk 6]
```diff
@@ -270,6 +272,7 @@ Setup will create infra resources and schedule daily security control scan on ta
 > **Note:**
 > 1. _Setup may take up to 5 minutes to complete._
 > 2. _For better performance, we recommend using one location for hosting central scanning user-assigned MI and resources which will be created in the following installation steps using the `Install-AzSKTenantSecuritySolution` cmdlet._
+> 3. _To install AzTS setup with **VNet integration**, uncomment switch `-EnableVnetIntegration` and then run the installation command `Install-AzSKTenantSecuritySolution`._ To know more about VNet Integration refer to this [FAQ](README.md#why-should-i-integrate-my-azts-solution-setup-with-vnet).
```
[the remark this answers is not in the document]
Comment (ritika-msft) [reply 01-Setup and getting started/README.md]: updated note.

File: 01-Setup and getting started/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (ritika-msft) [reply 01-Setup and getting started/README.md]: will update the Note part for vnet integration, after discussing with Vishal.

**Review by SINIKI — APPROVED** [review]

**Pull request merged**